jobs/env/ci-kubernetes-e2e-gce-scale-performance.env

```diff
@@ -5,6 +5,7 @@ CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
 NUM_NODES=5000
 NODE_SIZE=n1-standard-1
 NODE_DISK_SIZE=50GB
+MASTER_MIN_CPU_ARCHITECTURE=Intel Broadwell
```

Comment:
How is this file read? If just sourced by shell, then I'm afraid it will interpret it as: run command Broadwell with MASTER_MIN_CPU_ARCHITECTURE set to "Intel"

Reply:
I think that shouldn't be a problem.. the spaces are processed accordingly. For e.g see `CONTROLLER_MANAGER_TEST_ARGS` env var.
And IIUC this is the logic used:

https://github.com/kubernetes/test-infra/blob/0ac5871c022df47b34a0626ee34fbcd2e99f52f9/scenarios/kubernetes_e2e.py#L71-L73

which seems fine.

Reply:
The env file itself is read line-by-line like this:

https://github.com/kubernetes/test-infra/blob/0ac5871c022df47b34a0626ee34fbcd2e99f52f9/scenarios/kubernetes_e2e.py#L144-L151